docs/releases/1.9.txt

```diff
@@ -43,6 +43,10 @@ Minor features
   the old URL still redirects to the new one for backwards compatibility, but
   it may be removed in a future version.
 
+* In addition to ``ModelAdmin.list_select_related`` the method
+  ``ModelAdmin.get_list_select_related()`` was added to be consistent with
```

Comment:
Need to reference the full path because you're not in a `currentmodule`.
```
* :meth:`ModelAdmin.get_list_select_related()
  <django.contrib.admin.ModelAdmin.get_list_select_related>` was added to allow
  changing the ``select_related()`` values used in the admin's changelist query
  based on the request.
```